docs/BreakingChanges/087 WPF spawns a wisptis.exe process which can freeze .md

```diff
@@ -0,0 +1,25 @@
+## 87: WPF spawns a wisptis.exe process which can freeze the mouse
```

Comment:
{
  "Id": "87",
  "Title": "WPF spawns a wisptis.exe process which can freeze the mouse",
  "Details": "An issue was introduced in 4.5.2 that causes `wisptis.exe` to be spawned that can freeze mouse input.",
  "Suggestion": "A fix for this issue is available in a servicing release of the .NET Framework 4.5.2 (hotfix rollup 3026376), or by upgrading to the .NET Framework 4.6",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/927119/net-framework-4-5-2-causes-applications-using-wpf-to-spawn-a-wisptis-exe-process-which-can-freeze-the-mouse",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.2",
  "VersionFixed": "4.6",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 1
}

**Parsed APIs:**
- [ ] Not detectable via API analysis